examples/influxDB2-nifi/values.yaml

```diff
@@ -0,0 +1,243 @@
+---
+# Default values for FADI are defined here: https://github.com/cetic/helm-fadi.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+# Values you define here will overwrite default values from helm-fadi.
+
+spark:
+  enabled: false
+
+superset:
+  enabled: false
+
+minio:
+  enabled: false
+  persistence:
+    enabled: true
+    size: 50Gi
+
+
+grafana:
+  enabled: false
+  service:
```

Comment:
for clarity sake: when a service is not enabled, remove the rest of configuration